packages/os/settings-applier.service

```
@@ -12,4 +12,4 @@ RemainAfterExit=true
 StandardError=journal+console
 
 [Install]
-WantedBy=multi-user.target
+WantedBy=preconfigured.target
```

Previously, this unit was effectively `RequiredBy: configured.target`.
```suggestion
RequiredBy=preconfigured.target
```

---

Done